docs/schema/changelog.rst

```diff
@@ -15,6 +15,10 @@ ChangeLog
 [Unreleased] - 2021-03-30
 =========================
 
+Added
+-----
+- Support for describing the Public listing of an entity which is a Publicly Listed Company (PLC), using a new ``publicListing`` object.
```

Comment:
Could you edit this to:

"Support for describing the traded securities and status of a publicly listed company (PLC): a new ``publicListing`` object has been added to Entity Statements."

Thanks, @kindly.